mlflow/server/js/src/components/RunView.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .run-info {
-  min-width: 440px;
+  min-width: 450px;
```

Comment:
This change allows "Git commit" to be either on the left or right side -- without these extra 10 pixels, the git commit when source version is absent will push the entry on the same line to the right.